numba_dppy/tests/test_usm_ndarray_type.py

```diff
@@ -45,10 +44,7 @@ def usm_type(request):
     return request.param
 
 
-def test_usm_ndarray_type(offload_device, dtype, usm_type):
-    if skip_test(offload_device):
-        pytest.skip()
-
+def test_usm_ndarray_type(dtype, usm_type):
```

Comment:
Here is the regression: before - 54 tests, after - 30. But the **offload_device** were not used, so I removed them